### PR TITLE
changes to my userspace and preonic keymap ( spring 2019 )

### PR DESCRIPTION
**Pull request by kuatsure**
changes to my user and my preonic keymap files

## Description
getting prepped for planck that's coming soon

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: users/kuatsure/kuatsure.c [hunk 1]
```diff
@@ -1,35 +1,64 @@
 #include "kuatsure.h"
-
-void tmux_prefix(void) {
-  register_code(KC_LCTL);
-  register_code(KC_SPC);
-
-  unregister_code(KC_LCTL);
-  unregister_code(KC_SPC);
+#include "quantum.h"
```
Comment (drashna) [inline users/kuatsure/kuatsure.c]: This shouldn't be needed, since `kuatsure.h` includes it. 
```suggestion
```

File: users/kuatsure/kuatsure.c [hunk 2]
```diff
@@ -1,35 +1,64 @@
...
+#include "action.h"
```
Comment (drashna) [inline users/kuatsure/kuatsure.c]: This shouldn't be needed, since `quantum.h` includes it.

```suggestion
```

File: users/kuatsure/kuatsure.c [hunk 3]
```diff
@@ -1,35 +1,64 @@
...
-void tmux_pane_switch(uint16_t keycode) {
-  tmux_prefix();
-
-  register_code(KC_Q);
-  unregister_code(KC_Q);
-
-  register_code(keycode);
-  unregister_code(keycode);
+void tmux_prefix(void) {
+  SEND_STRING(SS_LCTRL(" "));
```
Comment (drashna) [inline users/kuatsure/kuatsure.c]: Alternatively, you could also do: 
```suggestion
  tap_code16(LCTL(KC_SPC));
```

**Comment by kuatsure** [comment]
@drashna changes requested are in now :) thanks :)

**Comment by drashna** [comment]
Thanks!

**Pull request merged**